client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts

```diff
@@ -11,6 +11,14 @@ export function useProcessingLoadingMessages() {
 
 	const stepData = useSelect( ( select ) => select( STEPPER_INTERNAL_STORE ).getStepData() );
 	switch ( stepData.intent ) {
+		case SiteIntent.Copy:
```

Comment:
@wojtekn Are we setting `site_intent` for the Simple site? I thought we were, but I don't see it anywhere...

Reply:
No, you may look for `setIntentOnSite`.

Reply:
I ended up directly referencing `flow` 851c6b91b2

`site_intent` is conceptually slightly different